_layouts/participate.html

```diff
@@ -9,6 +9,16 @@
 <div class="row">
 	<div class="col-md-8">
 		{{content}}
+		{% if page.location %}
```

Comment:
This seems a bit magic. I'm not necessarily opposed, but I wonder whether a simple

`{% include locations/westbeth.md | markdownify %}`

in the main page would be clearer? It would also leave the page order more flexible.

Thoughts?

Reply:
(Comment was re: that whole block)

Reply:
That is nice and simple. On the other hand, the intent of this PR was start moving towards a more declarative approach where we can define certain data in the front matter, and have a more consistent layout for all pages in the same category. In theory it would make it easier to make changes to the layout without having to update individual pages. I don't really mind though, if you prefer to keep it simple.

Reply:
Out of curiosity, I did try your suggestion locally, but it seems like it's not valid liquid syntax to add a pipe to an `include` statement in this way. 

```
  Liquid Exception: Invalid syntax for include tag: | markdownify Valid syntax: {% include file.ext param='value' param2='value' %} in fall2016.md
jekyll 3.8.5 | Error:  Invalid syntax for include tag:

  | markdownify

Valid syntax:

  {% include file.ext param='value' param2='value' %}
```

That's why I had to use the `assign` and `capture` approach in the layout. I think that would be too verbose to add to every individual page. I'm not that familiar with Jekyll, though, so maybe there's an easier way to do it that I'm not aware of.

Reply:
That makes a lot of sense. Thanks for talking through this with me, I'm into it!

Reply:
Not at all! Thanks!